applications/tari_base_node/src/commands/command/ban_peer.rs

```diff
@@ -0,0 +1,85 @@
+use std::time::Duration;
+
+use anyhow::Error;
+use async_trait::async_trait;
+use clap::Parser;
+use tari_app_utilities::utilities::UniNodeId;
+use tari_comms::peer_manager::NodeId;
+
+use super::{CommandContext, HandleCommand};
+use crate::LOG_TARGET;
+
+/// Bans a peer
+#[derive(Debug, Parser)]
+pub struct ArgsBan {
+    /// hex public key or emoji id
+    node_id: UniNodeId,
+    /// length of time to ban the peer for in seconds
+    #[clap(default_value_t = std::u64::MAX)]
+    lenght: u64,
```

Comment:
```suggestion
    length: u64,
```
Maybe `time_secs` is better but up to you